### ISSUE TITLE
[CLOSED] macOS move application file from home directory to Application Folder

### ISSUE THREAD

**Issue by tychobrailleur** [ho-dev/HO#5]
<a href="https://github.com/akasolace"><img src="https://avatars.githubusercontent.com/u/1136496?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [akasolace](https://github.com/akasolace)**
_Monday Oct 29, 2018 at 20:48 GMT_
_Originally opened as https://github.com/akasolace/HO/issues/5_

----

2 possibilities:
 - find a way to set chdir on os x startup script (does not look feasible)
 - move from relative to absolute path on mac os only (possible lot of work and very ennoying to do)

**Comment by tychobrailleur** [comment]
<a href="https://github.com/akasolace"><img src="https://avatars.githubusercontent.com/u/1136496?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [akasolace](https://github.com/akasolace)**
_Friday May 01, 2020 at 07:22 GMT_

----

@pierpaolocira  I was cleaning old issues and I found this one which is link or even similar to #372 
So I suggest I also assigned it to you for the day you will be able to tackle those ... let me know if that is ok with you.
cheers

**Comment by tychobrailleur** [comment]
<a href="https://github.com/akasolace"><img src="https://avatars.githubusercontent.com/u/1136496?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [akasolace](https://github.com/akasolace)**
_Thursday May 07, 2020 at 12:46 GMT_

----

@pierpaolocira the way I see it this is a duplicate of #372 so I am closing this one. If you disagree you can reopen it ... 
cheers

**Comment by tychobrailleur** [comment]
<a href="https://github.com/pierpaolocira"><img src="https://avatars.githubusercontent.com/u/11422667?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [pierpaolocira](https://github.com/pierpaolocira)**
_Friday May 08, 2020 at 22:43 GMT_

----

I do confirm... it's the same, although the final location is different. Here it is proposed to put it into the Application folder, but it doesn't sound as a good idea because it might be lost/overwritten easily (and MacOs uses another default location for such kind of information, which is what I used in 372, i.e. "application support"). It's a pity I have no time for proper testing, because it's ready since a long time... but I will feel confident to replease only after proper testing from me (and then from some voulunteer)...

**Comment by tychobrailleur** [comment]
<a href="https://github.com/akasolace"><img src="https://avatars.githubusercontent.com/u/1136496?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [akasolace](https://github.com/akasolace)**
_Sunday May 10, 2020 at 11:08 GMT_

----

@pierpaolocira I do agree we need proper testing first.  That would be a very nice improvement so really looking forward to it ...
